delegator/src/function/function.service.ts

```diff
@@ -0,0 +1,78 @@
+import { Injectable } from '@nestjs/common'
+import { Prisma } from '@prisma/client'
+import { PrismaService } from '../prisma.service'
+import { FunctionDto } from './dto/function.dto'
+import Caver from 'caver-js'
+
+@Injectable()
+export class FunctionService {
+  caver: any
+
+  constructor(private prisma: PrismaService) {
+    this.caver = new Caver(process.env.PROVIDER_URL)
+  }
+
+  encryptFunctionName(functionName: string) {
+    return this.caver.klay.abi.encodeFunctionSignature(functionName)
+  }
+
+  async create(functionDto: FunctionDto) {
+    const data: Prisma.FunctionUncheckedCreateInput = {
+      name: functionDto.name,
+      encodedName: this.encryptFunctionName(functionDto.name),
+      contractId: BigInt(functionDto.contractId)
```

Comment:
Is it necessary to explicitly convert it to `BigInt`? In definition of `FunctionDto` it is already defined as a `bigint`.

```
contractId: bigint
```

I think it should work without this explicit conversion. Can you please check it out?